.circleci/config.yml

```diff
@@ -10,12 +10,12 @@ jobs:
       - image: circleci/golang:1.10
```

Comment:
Should we update this image @camilamacedo86 - I believe `circleci/golang:1.12` is available for use

Reply:
done